packages/circuit-ui/components/Table/components/TableBody/TableBody.tsx

```diff
@@ -66,23 +64,15 @@ const TableBody = ({
         >
           {cells.map((cell, cellIndex) =>
             rowHeaders && cellIndex === 0 ? (
-              <Fragment key={`table-cell-${rowIndex}-${cellIndex}`}>
-                <TableHeader
-                  fixed
-                  condensed={condensed}
-                  scope="row"
-                  isHovered={sortHover === cellIndex}
-                  sortParams={{ sortable: false }}
-                  {...mapCellProps(cell)}
-                />
-                <TableCell
-                  role="presentation"
-                  condensed={condensed}
-                  aria-hidden="true"
-                >
-                  {getCellChildren(cell)}
-                </TableCell>
```

Comment:
I didn't understand the point of this duplicate, hidden `TableCell`. I assume it was necessary for the previous, hacky implementation of the sticky row header, so I removed it.